gfauto/gfauto/amber_converter.py

```diff
@@ -513,18 +517,19 @@ def get_amber_script_shader_def(shader: Shader, name: str) -> str:
 def graphics_shader_job_amber_test_to_amber_script(
     shader_job_amber_test: ShaderJobBasedAmberTest, amberfy_settings: AmberfySettings
 ) -> str:
-    # Guaranteed, and needed for type checker.
-    assert isinstance(shader_job_amber_test.variant, GraphicsShaderJob)  # noqa
 
     result = get_amber_script_header(amberfy_settings)
 
-    jobs = [shader_job_amber_test.variant]
+    jobs = shader_job_amber_test.variants
 
     if shader_job_amber_test.reference:
         assert isinstance(shader_job_amber_test.reference, GraphicsShaderJob)  # noqa
         jobs.insert(0, shader_job_amber_test.reference)
```

Comment:
I think this will modify the `shader_job_test` input. Maybe just do `jobs = [shader_job_amber_test.reference] + jobs` or something like that. Or ensure that jobs is a copy when first assigned (which I think you can do with a slice...maybe `jobs = shader_job_amber_test.variants[:]`)